src/client/locales/en.json

```diff
@@ -182,10 +182,12 @@
   "get_boost_content": "Want to boost your post? Add the topic \"busy\" and get a like from us.",
   "drafts": "Drafts",
   "drafts_empty": "You don't have any draft saved",
+  "drafts_description": "Posts that never made public. You can republish them or delete them.",
```

Comment:
What you mean by republish them? We should find a better sentence, it feel little bit confusing.

Reply:
I just made it exactly how it was on the mockup lol https://cdn.discordapp.com/attachments/371597897055076363/382511581055025152/Drafts.png

Reply:
updating the text now to `These are posts that were never made public. You can publish them or delete them.`